modules/bridge/src/main/ts/ephox/bridge/components/menu/CardMenuItem.ts

```diff
@@ -0,0 +1,48 @@
+import { FieldSchema, ValueSchema } from '@ephox/boulder';
+import { Fun, Optional, Result } from '@ephox/katamari';
+import { Title, TitleSpec } from './Title';
+import { CommonMenuItem, commonMenuItemFields, CommonMenuItemInstanceApi, CommonMenuItemSpec } from './CommonMenuItem';
+import { CardContainer, CardContainerSpec, itemSchema } from './CardContainer';
+import { Description, DescriptionSpec } from './Description';
+import { Image, ImageSpec } from './Image';
+
+export type ContainerItemSpec =
+  CardContainerSpec |
+  ImageSpec |
+  DescriptionSpec |
+  TitleSpec;
+
+export type ContainerItem =
+  CardContainer |
+  Image |
+  Description |
+  Title;
+
+export interface CardMenuItemInstanceApi extends CommonMenuItemInstanceApi { }
+
+export interface CardMenuItemSpec extends Omit<CommonMenuItemSpec, 'text'> {
+  type: 'cardmenuitem';
+  ariaLabel?: string;
```

Comment:
Why do we have `ariaLabel` here, as we don't include that on any other menu item? If we do need this, I believe this should instead be either be `label` or `tooltip` to keep consistent with the rest of the bridge spec/api.

Reply:
I thought "text" made little sense since it wont ever be put in the dom as a `tox-collection__item-label` item, which other menu items do, but I can change that.

Reply:
Yeah, I'd agree `text` doesn't make sense since it won't ever be visible. However `ariaLabel` isn't used anywhere else, that was more the point I was trying to make sorry.

Reply:
Speaking of this, does it make sense to include `shortcut` as well, given that also won't ever be visible?

Reply:
Great that you saw that, `shortcut` should be excluded as well.